tests/model_fields/tests.py

```diff
@@ -661,6 +689,14 @@ def test_backend_range_validation(self):
             }
             with self.assertRaisesMessage(ValidationError, expected_message):
                 instance.full_clean()
+            try:
+                with transaction.atomic():
+                    instance.save()
+                qs = self.model.objects.filter(value__lte=min_value)
+                self.assertEqual(qs.count(), 1)
+                self.assertEqual(qs[0].value, min_value)
+            except (DataError, IntegrityError):
```

Comment:
Will add a comment, some backends such as Postgres don't fail silently on integers that are out of range. In comparison MySQL and SQLite just munge the data. We check that our validators are correct by comparing the values we get from the database.